### PR TITLE
Create post

### PR DESCRIPTION
**Pull request by HardysJin**
### Check list

 - [x] The branch name follows the naming convention.
 - [x] Tests have been written.
 - [x] The code has a standard coding style or fixed using `yarn lint`.
 - [x] The pull request is attached in [Trello board](https://trello.com/b/pPtDOXkb).

### CODE REVIEW

**Comment by HardysJin** [comment]
[Create post](https://trello.com/c/wjGnITJ3/2-create-post)

File: src/tests/features/post.test.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { call } from "../utils/services";
+
+describe("PostResolver", () => {
+    it("Check createPost", async () => {
+        const createPostMutation = `mutation {
+            createPost(data: { content: "First post"}) {
```
Comment (senhungwong) [inline src/tests/features/post.test.ts]: First post can be pull out as a variable so the string is not repeated.

```
const content = "First post";

const createPostMutation = `mutation {
    createPost(data: { content: "${content}" }) {
        content
    }
}`

const res = await call({ source: createPostMutation });

expect(res).toMatchObject({
    data: {
        createPost: {
            content,
        },
    },
});
```

File: src/resolvers/PostResolver.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@Resolver()
+export class PostResolver {
+    @Mutation(returns => Post)
+    async createPost(
+        @Arg("data") input: PostInput,
+    ): Promise<Post> {
+        const post = await Post.create({
+            ...input,
+            createdAt: ~~(Date.now() / 1000),
```
Comment (senhungwong) [inline src/resolvers/PostResolver.ts]: `Date.now() | 1000` is faster in getting current timestamp.

Comment (HardysJin) [reply src/resolvers/PostResolver.ts]: Do you mean `Date.now() / 1000 | 0`?, `Date.now() | 1000` does not return correct timestamp.

Comment (senhungwong) [reply src/resolvers/PostResolver.ts]: Ohh, right. Then it doesn't matter

**Pull request merged**